### PR TITLE
New version: Microsoft.Promptflow version 1.8.0

### PR DESCRIPTION
**Pull request by YingChen1996**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148606)

### CODE REVIEW

**Comment by YingChen1996** [comment]
issue: #148628

**Review by mdanish-kh — APPROVED** [review]

File: manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Copyright: Copyright (C) Microsoft Corporation.
+CopyrightUrl: https://github.com/microsoft/promptflow/blob/main/LICENSE
+ShortDescription: Prompt flow is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring.
+Description: Prompt flow is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring. It makes prompt engineering much easier and enables you to build LLM apps with production quality.
+Moniker: pf
+Tags:
+- Command-Line
+- pf
+- pfazure
+InstallationNotes: Winget installs the 64-bit CLI on 64-bit OS by default now
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.locale.en-US.yaml:25]: This doesn't look like the right use case for `InstallationNotes`

File: manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+PrivacyUrl: https://go.microsoft.com/fwlink/?LinkId=521839
+Author: Microsoft Corporation
+PackageName: Microsoft Promptflow
+PackageUrl: https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2
+License: MIT License
+LicenseUrl: https://github.com/microsoft/promptflow/blob/main/LICENSE
+Copyright: Copyright (C) Microsoft Corporation.
+CopyrightUrl: https://github.com/microsoft/promptflow/blob/main/LICENSE
+ShortDescription: Prompt flow is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring.
+Description: Prompt flow is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring. It makes prompt engineering much easier and enables you to build LLM apps with production quality.
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.locale.en-US.yaml:19]: The extended description should not match the short description

File: manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Moniker: pf
+Tags:
+- Command-Line
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.locale.en-US.yaml:22]: Tags should be lowercase

File: manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+PackageIdentifier: Microsoft.Promptflow
+PackageVersion: 1.8.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: user
+ProductCode: '{3D44836A-5D49-4ABD-AE4F-CEDACF4339B3}'
+DisplayInstallWarnings: true
+AppsAndFeaturesEntries:
+- DisplayName: promptflow (64-bit)
+  UpgradeCode: 772aa21f-f8d4-4771-b910-1dbce3f1920c
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/Promptflow/1.8.0/Microsoft.Promptflow.installer.yaml:13]: Should be written as '{772AA21F-F8D4-4771-B910-1DBCE3F1920C}'

**Review by mdanish-kh — COMMENTED** [review]
Didn't want to block PR based on these nits, but I'll be fixing these in a separate PR

cc @YingChen1996

**Comment by mdanish-kh** [comment]
- PR created: https://github.com/microsoft/winget-pkgs/pull/148637

**Pull request merged**